packages/frontpage/app/(app)/_components/vote-button.tsx

```diff
@@ -42,7 +42,7 @@ export function VoteButton({
       <Button
         variant="ghost"
         size="icon"
-        className="hover:bg-gray-100 dark:hover:bg-gray-800 z-10 relative group"
+        className="z-10 relative group"
```

Comment:
We can move these styles to inside the button component i think

Reply:
@tom-sherman not sure `z-10 relative group` are even needed on the button? They dont seem to do anything

Reply:
Yeah for sure. Think this was from when I was trying to make the whole card clickable with an overlaid `<a>`.